pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -342,6 +342,12 @@ func TestPriorityQueue_Update(t *testing.T) {
 		t.Error("Expected unschedulablePods to be 1.")
 	}
 	updatedPod = medPriorityPodInfo.Pod.DeepCopy()
+
+	q.Update(medPriorityPodInfo.Pod, updatedPod)
```

Comment:
what are we testing here? can you add a comment?

Reply:
or better, use `t.Log` to indicate what conditions changed.
If you can, add similar notes for the existing tests.

Arguably, these tests are very old and need some love.